src/components/SelectCountry/CountryDropDown.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from "react";
+
+// CountryDropDownProps 타입 지정
+type CountryDropDownProps = {
+  countries: string[];
+  showDropDown: boolean;
+  toggleDropDown: Function;
+  countrySelection: Function;
+};
+
+const CountryDropDown: React.FC<CountryDropDownProps> = ({
+  countries,
+  countrySelection,
+}: CountryDropDownProps): JSX.Element => {
+  const [showDropDown, setShowDropDown] = useState<boolean>(false);
+
+  const onClickHandler = (country: string): void => {
+    countrySelection(country);
+  };
+
+  useEffect(() => {
+    setShowDropDown(showDropDown);
+  }, [showDropDown]);
+
+  return (
+    <>
+      <div className={showDropDown ? "dropdown" : "dropdown active"}>
+        {countries.map((country: string, index: number): JSX.Element => {
```

Comment:
이미 countries가 string배열이기 때문에 타입을 명시 안해도 될 것 같아요
```suggestion
     {countries.map((country, index): => {
```

Reply:
넵.. 따로 또 지정해줘야 되는 줄 알았습니다